tests/test_run.py

```diff
@@ -36,13 +36,15 @@ def test_a2c(env_id):
 
 @pytest.mark.parametrize("env_id", ["CartPole-v1", "Pendulum-v1"])
 @pytest.mark.parametrize("clip_range_vf", [None, 0.2, -0.2])
-def test_ppo(env_id, clip_range_vf):
+@pytest.mark.parametrize("normalize_advantage", [False, True])
```

Comment:
I will write a more simple test tomorrow for A2C and PPO, as this creates way too many tests (it doubles its number) for just toggling `normalize_advantage` ;)